lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -316,12 +336,14 @@ const cli = yargs
     'expectations-path': 'The path to the expected audit results file',
     'debug': 'Save the artifacts along with the output',
   })
-  .require('config-path')
-  .require('expectations-path')
+  .require('config-path', true)
```

Comment:
are our typedefs for yargs off or was this really required before and we were missing it?

Reply:
yeah they are off. we use a DefinitelyTyped which doesn't define a few things. This `require` is apparently a deprecated API, perhaps that is why.